source/localizable/tutorial/installing-addons.md

```diff
@@ -18,7 +18,11 @@ Run the following command to install the addon:
 ember install ember-cli-tutorial-style
 ```
 
-Starting the server will incorporate the new CSS and refreshing our browser window will give you this:
+Since Ember addons are regular npm packages, `ember install` installs them in
+the `node_modules` directory, and makes an entry in `package.json`. Be sure to
```

Comment:
See CONTRIBUTING.md for line break guidance: 

When writing prose:

Use a single space after periods.
Manually break lines:
at the beginning of every new sentence;
after the last comma in sentences over 120 characters; or
after logical statements in sentences over 120 characters with no commas.